packages/web-vite/src/components/LibraryItemCard.tsx

```
@@ -182,12 +189,29 @@ const LibraryItemCard: React.FC<LibraryItemCardProps> = ({
+        >
           <button
             className="card-menu-item"
-            onClick={() => handleMenuAction(item.state === 'ARCHIVED' ? 'unarchive' : 'archive')}
+            onClick={() =>
+              handleMenuAction(
+                item.state === 'ARCHIVED' ? 'unarchive' : 'archive'
+              )
+            }
           >
-            <svg width="16" height="16" viewBox="0 0 24 24" fill="none" stroke="currentColor" strokeWidth="2" strokeLinecap="round" strokeLinejoin="round">
+            <svg
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)